assets/src/edit-story/output/utils/styles.js

```diff
@@ -27,6 +27,13 @@ function isHexColorString(s) {
 function CustomStyles() {
   const safeToFullRatio = PAGE_RATIO / FULLBLEED_RATIO;
   const fullToSafeRatio = 1 / safeToFullRatio;
+  const fullBleedHeight = FULLBLEED_RATIO / 2;
+  const safeZoneHeight = SAFE_ZONE_HEIGHT;
```

Comment:
No need to duplicate this. Just use `SAFE_ZONE_HEIGHT` directly.

Reply:
It was my suggestion to sanitize the constant here just to be safe - like for the other variables. Since we're using `dangerouslySetInnerHTML` here.

Reply:
Fixed with another calculation method.